launch/sim.launch.py

```diff
@@ -1,17 +1,37 @@
-import launch
+import os
+from ament_index_python.packages import get_package_share_directory
 from launch import LaunchDescription
 
-import launch_ros.actions
+from launch.actions import IncludeLaunchDescription, SetEnvironmentVariable, Shutdown
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch_ros.actions import Node
 
 
 def generate_launch_description():
-    return LaunchDescription([
-        launch_ros.actions.Node(package='rj_robocup',
-                                node_executable='grSim',
-                                arguments=['--headless']),
-        launch_ros.actions.Node(package='rj_robocup',
-                                node_executable='soccer',
-                                output='screen',
-                                arguments=['-b', '-sim'],
-                                on_exit=launch.actions.Shutdown()),
-    ])
+    bringup_dir = get_package_share_directory('rj_robocup')
+    launch_dir = os.path.join(bringup_dir, 'launch')
+
+    stdout_linebuf_envvar = SetEnvironmentVariable(
+        'RCUTILS_CONSOLE_STDOUT_LINE_BUFFERED', '1')
+
+    grsim = Node(package='rj_robocup',
+                 executable='grSim',
+                 arguments=['--headless'])
+
+    soccer = Node(package='rj_robocup',
+                  executable='soccer',
+                  output='screen',
+                  arguments=['-b', '-sim'],
+                  on_exit=Shutdown())
+
+    config_server = Node(package='rj_robocup',
+                         executable='config_server',
+                         output='screen',
+                         on_exit=Shutdown())
+
+    vision_receiver = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            os.path.join(launch_dir, "vision_receiver.launch.py")))
```

Comment:
Are long term nodes going to all be built this way?

Reply:
What do you mean by that?

Reply:
Basically, for the receiver, you have a second file, but for the other ones, you just build it in this file

Reply:
Oh. I think long term we're going to want to split it up into separate files and then include them, so that we can compose them in different configurations and whatnot